codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -99,6 +103,12 @@ public void removeStream(Http2Stream stream) {
 
     @Override
     public void addListener(Listener listener) {
+        // Ensure we don't add the same listener twice.
```

Comment:
I think we not need this. Fir example we not do this kind of check in our future impl as well